app/uk/gov/hmrc/vatapi/resources/VatReturnsResource.scala

```diff
@@ -63,16 +68,21 @@ object VatReturnsResource extends BaseResource {
       }
       }
     }
+
+    result.recover {
+      case ex => logger.warn(ex.getMessage)
+        InternalServerError(Json.toJson(Errors.InternalServerError))
```

Comment:
Need to change the logging message to follow our agreed logging format:

logger.warn(s"[VatReturnsResource] [submitVatReturn] Unexpected downstream error thrown ${ex.getMessage}")